src/txn/txn_rollback_to_stable.c

```diff
@@ -552,11 +554,11 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
 }
 
 /*
- * __rollback_abort_col_ondisk_kv --
- *     Fix the on-disk col version according to the given timestamp.
+ * __rollback_abort_ondisk_kv --
+ *     Fix the on-disk row K/V version according to the given timestamp.
```

Comment:
Remove `row` from the function description.

Reply:
Fixed.